tests/manuscripts/versionHistory.test.ts

```
@@ -643,18 +643,31 @@ describe("Manuscript Version History", () => {
+              data: null,
+              error: { message: "Not found", code: "PGRST116" },
+            }),
+          };
+        }
+        return {
+          select: jest.fn().mockReturnThis(),
+          eq: jest.fn().mockReturnThis(),
+          order: jest.fn().mockReturnThis(),
+          limit: jest.fn().mockReturnThis(),
+          lt: jest.fn().mockReturnThis(),
+          then: jest.fn().mockResolvedValue({
```

Biome (2.3.13)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)